webapp/www/templates/client_setup.html

```diff
@@ -26,20 +26,22 @@ <h2>Thank you for participating - only two steps left!</h2>
                   </g>
               </g>
           </svg>
-        <h4>Install the {{base_app}} app</h4>
+        <h4>Install the e-mission app</h4>
         </div>
         <div class="row responsive-sm">
-          <a class="col" href='{{android_base_app}}'><img style="height:75px;" alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png'/></a>
-          <qrcode class="col" aria-label="qrcode for google play installation" data="{{android_base_app}}" size="200"></qrcode>
+          <a class="col" href='https://play.google.com/store/apps/details?id=edu.berkeley.eecs.embase'><img style="height:75px;" alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/images/generic/en_badge_web_generic.png'/></a>
+          <qrcode class="col" data="https://play.google.com/store/apps/details?id=edu.berkeley.eecs.emission" size="200"></qrcode>
```

Comment:
there's an inconsistency there - the link is to emTripLog and the qrcode is to emission

Reply:
fixed this